examples/src/main/scala/org/apache/spark/examples/sql/hive/SparkHiveExample.scala

```diff
@@ -102,8 +101,63 @@ object SparkHiveExample {
     // |  4| val_4|  4| val_4|
     // |  5| val_5|  5| val_5|
     // ...
-    // $example off:spark_hive$
 
+    /*
+     * Save DataFrame to Hive Managed table as Parquet format
+     * 1. Create Hive Database / Schema with location at HDFS if you want to mentioned explicitly else default
+     * warehouse location will be used to store Hive table Data.
+     * Ex: CREATE DATABASE IF NOT EXISTS database_name LOCATION hdfs_path;
+     * You don't have to explicitly give location for each table, every tables under specified schema will be located at
+     * location given while creating schema.
+     * 2. Create Hive Managed table with storage format as 'Parquet'
+     * Ex: CREATE TABLE records(key int, value string) STORED AS PARQUET;
+     */
+    val hiveTableDF = sql("SELECT * FROM records").toDF()
+    hiveTableDF.write.mode(SaveMode.Overwrite).saveAsTable("database_name.records")
+
+    /*
+     * Save DataFrame to Hive External table as compatible parquet format.
+     * 1. Create Hive External table with storage format as parquet.
+     * Ex: CREATE EXTERNAL TABLE records(key int, value string) STORED AS PARQUET;
```

Comment:
it's weird to create an external table without a location. User may be confused between the difference between managed table and external table.

Reply:
@cloud-fan we'll keep all comments description at documentation with user friendly lines. I have added location also.